src/components/ManifestListItem.js

```diff
@@ -119,7 +122,15 @@ export class ManifestListItem extends React.Component {
             </Grid>
 
             <Grid item xs={4} sm={2}>
-              <WindowIcon className={ns('manifest-list-item-logo')} manifestId={manifestId} />
+              <Img
```

Comment:
Is it possible to use our own `CanvasThumbnail` class also to do something similar?

Reply:
I'm not sure what you mean?

Reply:
Nope it doesn't thanks!